extensions/ql-vscode/src/query-history.ts

```diff
@@ -926,7 +951,7 @@ export class QueryHistoryManager extends DisposableObject {
     }
 
     // Summary log file doesn't exist.
-    if (finalSingleItem.evalLogLocation && fs.pathExists(finalSingleItem.evalLogLocation)) {
+    if (finalSingleItem.evalLogLocation && await fs.pathExists(finalSingleItem.evalLogLocation)) {
```

Comment:
We should have had a linter error because of this. Good catch.

Reply:
Thanks!! Didn't notice this either.